### DISCUSSION TITLE
enable/disable kick function not available?

### DISCUSSION THREAD

**Discussion by Showmaster3000** [scribble-rs/scribble.rs#327] [General]
Hi, I just noticed that you cannot enable/disable the kick function anymore? At least at the sample page scribblers.fly.dev
Some months ago I'm sure it was possible to switch in the room settings. Did I miss something or is it a bug/was it accidentially removed?

**Comment by Bios-Marcel** [comment]
I decided to remove the ability to disable it. I figured if you don't need it, you just wouldn't use it. Since I was also rewriting the frontend, I was trying to simplify and slim down stuff.
